certified-connectors/LnkBio/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Lnk.Bio Connector
+This is a connector for Lnk.Bio. Lnk.Bio is a link-in-bio and website builder web tool and mobile app favoured by 600,000 creators. 
```

Comment:
@andreaolivato all files should be free of grammatical errors and typos. Please change "favoured" to "favored".